### DISCUSSION TITLE
Whether the log plug-in can add page turning function?

### DISCUSSION THREAD

**Discussion by MicroOps-cn** [grafana/grafana#51684] [Dashboards, Panels & Visualizations]
Similar to retrieving Loki data sources in explore, there is an "Older Logs" function, which allows you to view earlier logs

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
